storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.hpp

```diff
@@ -76,4 +77,11 @@ Uint32 ExponentialDelayWithJitter(Uint32 retry, Uint32 initialDelayInMS, Uint32
  */
 bool UnloadSchema(RS_Status status);
 
+
+/**
+ * Handle NDB schema releated errors, such as, Invalid schema errors
+ * This unloads the tables schema from the NDB::Dictionary
```

Comment:
unloads the schema of the table from ...

Reply:
fixed